build_rpm.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+git checkout -- setup.cfg
+rm -rf build dist *.egg *.egg-info
+
+# Build community source tarball
+
+python setup.py sdist --formats=gztar
+
+# Build F5 rpm spec
+
+/bin/cp -f f5-setup.cfg setup.cfg
+
+python setup.py bdist_rpm --spec-only \
+  --release=1 \
+  --provides=f5-neutron-lbaas-dashboard \
+  --packager="Qin Zhao <q.zhao@f5.com>" \
+  --post-install=rpm_post_install.sh
+
+# Repackage source tarball
+
+cd dist
```

Comment:
use ( ) to start a sub shell for the work.. changing directory in shell script is not convenient when we want to add more logic, we have to glance over the whole script to see where(which directory) I'm now.